### PR TITLE
Adds an enum Routes to variables Routing

### LINKED ISSUE #184: Add a route enum to variables
Lets add a route enum to [variables](https://github.com/HandsOnLearning/the-project/blob/master/src/variables.ts).

Right now we are doing this
```jsx
<Link to="/settings" />
<Route path="/settings" />
```

but we should have a single source of truth for routes, and variables would be a suitable place. Add an [enum](https://www.typescriptlang.org/docs/handbook/enums.html) to variables for routes, that'd enable us to do this:
```jsx
<Link to={Routes.Settings} />
<Route path={Routes.Settings} />
```

### PR DESCRIPTION
**Pull request by limbovski**
Closes #184

### CODE REVIEW

File: src/variables.ts [hunk 1]
```diff
@@ -7,3 +7,11 @@ export enum Color {
...
   GreySeeThru = "hsla(0,0%,100%,.3)",
 }
+
+// Some Routes below are set to "/" as placeholders
+// until we build the end points and replace them.
+export enum Routes {
+  Home = "/",
+  SignOut = "/",
+  Settings = "/settings",
+}
```
Comment (MoSattler) [inline src/variables.ts:17]: to be honest, I think we should remove `Signout` right now (as a route and as part of the expanded list) until we actually implement it. But that'd be another issue.

**Comment by MoSattler** [comment]
@SimonNord https://www.typescriptlang.org/docs/handbook/enums.html

**Comment by SimonNord** [comment]
> @SimonNord https://www.typescriptlang.org/docs/handbook/enums.html

Thanks, I read up a bit on that exact page =)

File: src/variables.ts (hunk 1 above)
Comment (limbovski) [reply src/variables.ts:17]: Ok will do!

**Pull request merged**